### DISCUSSION TITLE
Persona 5 The Royal dosen't boot.

### DISCUSSION THREAD

**Discussion by ionelush2001** [InoriRus/Kyty#26] [Show and tell]
![Persona 5 The Royal 1](https://user-images.githubusercontent.com/60799365/170826801-6626fe09-d6de-44b5-8ce8-9f202d013a1a.jpg)
![Persona 5 The Royal 4](https://user-images.githubusercontent.com/60799365/170827078-a242fafd-1f5f-4b81-b4c6-284b0d98608c.jpg)
![Persona 5 The Royal 3](https://user-images.githubusercontent.com/60799365/170826802-405bcaa7-00b7-4b42-b57b-64dab5083453.jpg)

**Comment by DomoHeishda** [comment]
Are you using windows? Also where did you get your iso for the game?

**Reply by GamingDucking** [reply]
piracy moment

**Comment by gandalfthewhite19890404** [comment]
Does not boot due unimplemented functions. In particular case of crash - FSin. There are many other unimplemented functions, so game gonna crash due another function even if FSin will be implemented.
